tests/network/bindingplugin_passt.go

```diff
@@ -63,12 +64,18 @@ var _ = SIGDescribe("[Serial] VirtualMachineInstance with passt network binding
 
 	BeforeEach(func() {
 		const passtBindingName = "passt"
+
+		var passtComputeMemoryOverheadWhenAllPortsAreForwarded = resource.MustParse("800Mi")
```

Comment:
Are we sure it is 800M and not 500M as Passt tests declare?

Reply:
Changed to 500Mi after an offline discussion with @sbrivio-rh.